blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,606 @@
+
+Here is even more benchmarks:
+
+<Image src={require("./img/benches/13526-Sly1-pt1.webp").default} />
+
+<Image src={require("./img/benches/13526-UG2-pt2.webp").default} />
+
+<Image src={require("./img/benches/13526-MW-pt3.webp").default} />
+
+<Image src={require("./img/benches/13526-CRB-pt4.webp").default} />
+
+<Image src={require("./img/benches/13651-SCM-pt5.webp").default} />
+
+<Image src={require("./img/benches/13651-SCM-pt6.webp").default} />
+
+<Image src={require("./img/benches/13651-SCM-pt7.webp").default} />
+
+<PCSX2PRLink authors="lightningterror" prNums="13549,13575,13696,13525">
+  ### GS/Direct3D11/12: Destination Alpha Testing Optimizations
+</PCSX2PRLink>
+
+Following the addition of software blending on Direct3D11/12, we can optimize some features where if we already use software blending we can use software destination alpha testing as well in shader which is faster instead of splitting it in multiple passes as it will be slower. We don't need to do it in multiple passes as we already have texture copies enabled so we can reuse them for destination alpha testing and bypassing the multiple passes originally needed.
+
+Some optimizations are missing on Direct3D12 like [#13525](https://github.com/PCSX2/pcsx2/pull/13525) but will be implemented during the v2.7 dev cycle.
+
+<Image src={require("./img/benches/13549-Enth-pt1.webp").default} />
+
+<PCSX2PRLink authors="TJnotJT,refractionpcsx2" prNums="13142,13138,13005,13685">
+  ### GS/HW: Shuffle improvements
+</PCSX2PRLink>
+
+Several fixes have been made to texture shuffle emulation in order to fix some long-standing games.
+
+#### Handles Texture Shuffle With Pixel Reversals ([#13142](https://github.com/PCSX2/pcsx2/pull/13142))
+
+Detect when a game (Colin McRae Rally 2005) is doing a texture shuffle and (a) flipping pixel locations horizontally in columns and (b) correcting the pixel locations by flipping pixels horizontally again. Ignore both flips so that they cancel out. To keep it short what this does is fix Colin McRae Rally and is now playable with no hardware rendering issues.
+
+<SliderCompare
+  before={require("./img/comparisons/13142-CMR-before.webp").default}
+  after={require("./img/comparisons/13142-CMR-after.webp").default}
+/>
+
+#### Handles Texture Shuffle Sprite Not 16-pixel-multiple Wide ([#13138](https://github.com/PCSX2/pcsx2/pull/13138))
+
+Do special handling for texture shuffling sprites that are not 8 pixels wide or a multiple of 16 pixels wide. Align such sprites to 16 pixel boundaries horizontally. This fixes some issues on Alter Echo.
+
+#### Add CRC for Sand Grain Games Palette Shuffle Effect ([#13005](https://github.com/PCSX2/pcsx2/pull/13005))
+
+We have no way to do this effect directly without having 50-100 readbacks per frame, which is very very slow, this emulates the effect they're trying to achieve (G->A shuffle with palette).
+
+This fixes some long standing issues with Cabela's games, image comparisons below:
```

Comment:
"long-standing" was hyphenated just above, now it isn't. Should pick one or the other and normalize it through the whole post.